packages/atomic/src/components/commerce/facets/facet-number-input/atomic-commerce-facet-number-input.tsx

```diff
@@ -0,0 +1,140 @@
+import {NumericFacet} from '@coveo/headless/commerce';
```

Comment:
For this one, we should definitely refactor and share a lot more code with `atomic-facet-number-input`.

We just need to make sure there is no reliance on any `@coveo/headless` type (just primitive) and there's no reason we cannot use the same input for both type of facets.